### ISSUE TITLE
MIT Voiture ?

### ISSUE THREAD

**Issue by Clemog** [datagir/monimpacttransport#120]
Suite à #117, je me dis qu'une utilisation possible de MIT pourrait être de comparer uniquement différent types de voitures avant un achat par exemple. MyCO2 permet de choisir directement son véhicule via la marque/modèle, on pourrait imaginer un moyen pour que l'utilisateur spécifie son véhicule et le compare à d'autres .. (C'est une première réflexion peut etre qu'il y avait déjà une issue dessus :) )

**Comment by florianpanchout** [comment]
Yes, on attend une réponse du service transport pour proposer cet affichage :)
